handlers/emailer.js

```diff
@@ -1,32 +1,55 @@
 const nodemailer = require('nodemailer');
-const jwt = require('jsonwebtoken');
+const { google } = require("googleapis");
+const OAuth2 = google.auth.OAuth2;
 
 async function getTransporter() {
-  // If we have gmail credentials, use gmail to send the email
-  if (process.env.GMAIL_ADDRESS) {
+  if (process.env.GMAIL_CLIENT_ID) {
+
+    const oauth2Client = new OAuth2(
+      process.env.GMAIL_CLIENT_ID,
+      process.env.GMAIL_CLIENT_SECRET,
+      "https://developers.google.com/oauthplayground"
+    );
+    oauth2Client.setCredentials({refresh_token: process.env.GMAIL_REFRESH_TOKEN});
+
+    // const tokenRefreshRes = await oauth2Client.refreshAccessToken();
+    // const accessToken = tokenRefreshRes.credentials.access_token;
+
     return nodemailer.createTransport({
-      service: 'gmail',
+      host: 'smtp.gmail.com', port: 465, secure: true,
```

Comment:
Looked into the nodemailer docs, that's unfortunate that they lost their nice gmail service wrapper, would have been cool to seem them just update it to handle a lot of this oauth stuff for us